run.py

```diff
@@ -430,59 +479,34 @@ def run(args):
             distro.append(image_name.replace(".iso", ""))
 
         # get COMPOSE ID and ceph version
-        id = requests.get(base_url + "/COMPOSE_ID")
-        compose_id = id.text
-
-        if "rhel" == inventory.get("id"):
-            ceph_pkgs = requests.get(base_url + "/compose/Tools/x86_64/os/Packages/")
-            m = re.search(r"ceph-common-(.*?).x86", ceph_pkgs.text)
-            ceph_version.append(m.group(1))
-            m = re.search(r"ceph-ansible-(.*?).rpm", ceph_pkgs.text)
-            ceph_ansible_version.append(m.group(1))
-            log.info("Compose id is: " + compose_id)
-        else:
-            ubuntu_pkgs = requests.get(
-                ubuntu_repo + "/Tools/dists/xenial/main/binary-amd64/Packages"
-            )
-            m = re.search(r"ceph\nVersion: (.*)", ubuntu_pkgs.text)
-            ceph_version.append(m.group(1))
-            m = re.search(r"ceph-ansible\nVersion: (.*)", ubuntu_pkgs.text)
-            ceph_ansible_version.append(m.group(1))
+        if build != "released":
```

Comment:
Could we exempt `cvp` also? When the type is `cvp`, we would be using the packages available in the production mirror.

Reply:
Updated.